porting/first-steps.rst

```diff
@@ -57,11 +57,12 @@ Arch
 ^^^^^^^^^^
 
 If you have a pure ``amd64`` running you need to add ``[multilib]`` repository to your ``/etc/pacman.conf`` . This will allow you to install and run ``i686`` packages. Please refer to `<https://wiki.archlinux.org/index.php/multilib>`_
+Assuming you have already installed ``base-devel`` meta package; if not please install ``base-devel``.
 
 Install the required dependencies from official repository or AUR::
 
-   lineageos-devel git repo flex qemu binfmt-support qemu-user-static e2fsprogs sudo simg-tools
-      android-sdk-platform-tools bootimg-tools-git
+   lineageos-devel qemu binfmt-support qemu-user-static simg-tools
```

Comment:
`qemu` is not required iirc. are you sure that you need it?

Reply:
i would also appreciate if you specified which packages come from AUR and which packages come from official repos. an example installing with `pacaur` would be brilliant

Reply:
@abhishek9650 i am waiting for your comments on that one